src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -485,7 +485,9 @@ void GetCoolingPanelInput(EnergyPlusData &state)
         }
 
         thisCP.ColdSetptSchedName = s_ipsc->cAlphaArgs(7);
-        if ((thisCP.coldSetptSched = Sched::GetSchedule(state, thisCP.ColdSetptSchedName)) == nullptr) {
+        if (s_ipsc->lAlphaFieldBlanks(7)) {
+            thisCP.coldSetptSched = Sched::GetScheduleAlwaysOff(state);
+        } else if ((thisCP.coldSetptSched = Sched::GetSchedule(state, thisCP.ColdSetptSchedName)) == nullptr) {
```

Comment:
This will avoid the fatal when it's blank.

Question left is whether it should be AlwaysOff or AlwaysOn.

Reply:
https://github.com/NREL/EnergyPlus/blob/e7ecb2d53b6b2f4666bf5a56fc9b3ba35cf34190/src/EnergyPlus/ChilledCeilingPanelSimple.hh#L109

https://github.com/NREL/EnergyPlus/blob/e7ecb2d53b6b2f4666bf5a56fc9b3ba35cf34190/src/EnergyPlus/ScheduleManager.hh#L72

Reply:
All good!